pgctl/service.py

```diff
@@ -150,16 +165,36 @@ def ensure_exists(self):
         if not self.path.check(dir=True):
             raise NoSuchService("No such service: '%s'" % bestrelpath(str(self.path)))
 
-        # ensure symlink {service_dir}/supervise -> {scratch_dir}/supervise
-        # this will re-connect the service to its state descriptors if the symlinks have been deleted or moved
-        supervise_in_scratch = self.scratch_dir.join('supervise')
-        supervise_in_scratch.ensure_dir()
-        symlink_if_necessary(supervise_in_scratch, self.path.join('supervise'))
+        self._ensure_supervise_is_scratch('supervise')
         self.__exists = True
 
     def ensure_logs(self):
         self.ensure_exists()
-        self.path.ensure('log')
+        self.path.ensure_dir('logs')
+        self.path.join('logs').ensure('current')
+
+        self.path.ensure_dir('log')
+        with open(self.path.join('log', 'run').strpath, 'w') as log_run:
+            log_run.write('#!/bin/bash\n')
```

Comment:
what about adding a giant header as a warning to users?

```bash
#!/bin/bash
#####################################################################
# This file is automatically generated by pgctl.
# You should *NOT* modify this file or check it in to source control.
#####################################################################
```

you can be creative with what it actually says...